pkgs/os-specific/linux/open-iscsi/default.nix

```
@@ -1,36 +1,38 @@
-{ stdenv, fetchFromGitHub, automake, autoconf, libtool, gettext, utillinux, openisns, openssl, kmod }:
+{ stdenv, fetchFromGitHub, automake, autoconf, libtool, gettext,
+utillinux, openisns, openssl, kmod, perl, systemd, pkgconf }:
```

```suggestion
{ stdenv, fetchFromGitHub, automake, autoconf, libtool, gettext
, utillinux, openisns, openssl, kmod, perl, systemd, pkgconf 
}:
```
See https://nixos.org/nixpkgs/manual/#sec-syntax